tests/integration/queries/test_submission.py

```diff
@@ -0,0 +1,26 @@
+from pathlib import Path
+
+from indico.client import IndicoClient
+from indico.queries import (
+    JobStatus,
+    ListSubmissions,
+    ListWorkflowsForDataset,
+    WorkflowSubmission,
+)
+from indico.types import Job, ModelGroup
+from ..data.datasets import airlines_dataset, airlines_model_group
+
+
+def test_list_submissions(indico, airlines_dataset, airlines_model_group: ModelGroup):
```

Comment:
were you able to run this test against dev-repair?
credentials and API token pinned to the project-repair slack channel. you can also write tests for CreateSubmissionResult by going after any submission in the PENDING_REVIEW state - lmk and I can get you some ids you can test with:
workflow 1, submission 210
workflow 3, submission 205

Reply:
I did test on repair for list submissions, ahven't yet tested for createSubmissionResult

Reply:
sounds like, based on the conversation today, integration tests for SubmissionResult will be in the integration-tests folder. so no need here